internal/pkg/cli/task_run.go

```diff
@@ -437,22 +437,21 @@ func BuildTaskRunCmd() *cobra.Command {
 	}
 	cmd := &cobra.Command{
 		Use:   "run",
-		Short: "Run a one-off task",
-		Long:  `Run a one-off task with configurations such as cpu-units, memory, image, etc.`,
+		Short: "Run a one-off task on Amazon ECS.",
 		Example: `
-Run a task with default setting. You will be prompted to specify a task group name and an environment for the tasks to run in.
+Run a task with default settings. 
+You will be prompted to specify a task group name and an environment for the tasks to run in.
 /code $ copilot task run
 Run a task named "db-migrate" in the "test" environment under the current workspace.
 /code $ copilot task run -n db-migrate --env test
-Starts 4 tasks with 2GB memory, run a particular image, and run with a particular task role.
-/code $ copilot task run --num 4 --memory 2048 --image=python --task-role migrate-exec-role
+Run 4 tasks with 2GB memory, an existing image, and a custom task role.
+/code $ copilot task run --num 4 --memory 2048 --image=rds-migrate --task-role migrate-role
 Run a task with environment variables.
 /code $ copilot task run --env-vars name=myName,user=myUser
 Run a task with subnets and security groups.
```

Comment:
```suggestion
Run a task using the current workspace with specific subnets and security groups.
```

What do you think about this change, to make it clearer what `task run` will infer about the image?

Reply:
Done!